contracts/v2/PolygonZkEVMBridgeV2.sol

```diff
@@ -30,16 +30,16 @@ contract PolygonZkEVMBridgeV2 is
     }
 
     // bytes4(keccak256(bytes("permit(address,address,uint256,uint256,uint8,bytes32,bytes32)")));
-    bytes4 private constant _PERMIT_SIGNATURE = 0xd505accf;
+    bytes4 internal constant _PERMIT_SIGNATURE = 0xd505accf;
 
     // bytes4(keccak256(bytes("permit(address,address,uint256,uint256,bool,uint8,bytes32,bytes32)")));
-    bytes4 private constant _PERMIT_SIGNATURE_DAI = 0x8fcbaf0c;
+    bytes4 internal constant _PERMIT_SIGNATURE_DAI = 0x8fcbaf0c;
 
     // Mainnet identifier
-    uint32 private constant _MAINNET_NETWORK_ID = 0;
+    uint32 internal constant _MAINNET_NETWORK_ID = 0;
 
     // ZkEVM identifier
-    uint32 private constant _ZKEVM_NETWORK_ID = 1;
+    uint32 internal constant _ZKEVM_NETWORK_ID = 1;
```

Comment:
i think this both variables are not mandatory to be changed, review and do not update them if not mandatory
also applies `_MAX_LEAFS_PER_NETWORK`

Reply:
You are right that `_MAINNET_NETWORK_ID` and `_ZKEVM_NETWORK_ID` can remain private but others are being used at function overriding from sovereignBridge contract